services/apps/alcs/src/alcs/card/card-type/card-type.entity.ts

```diff
@@ -13,7 +13,9 @@ export enum CARD_TYPE {
   NOTIFICATION = 'NOTI',
 }
 
-@Entity()
+@Entity({
+  comment: '',
```

Comment:
Is it required to have an empty comment?

Reply:
Removed